backend/sozisel/test/support/factory.ex

```diff
@@ -46,4 +46,30 @@ defmodule Sozisel.Factory do
       session_template_id: attrs[:session_template_id] || insert(:template).id
     }
   end
+
+  def event_factory(attrs) do
+    %Event{
+      name: attrs[:name] || sequence(:name, &"Event name no. #{&1}"),
+      start_minute: attrs[:start_minute] || sequence(:start_minute, [5, 10, 15, 20, 25, 33]),
+      event_type:
+        attrs[:event_type] ||
+          %Quiz{
```

Comment:
this will probably need a refactor once we introduce more event types. Can you just please move this quiz generation to separate function? Not necessarily to a factory as it can't be directly inserted to database but to something like `quick_event_type`